nodejs_container/native/src/container.rs

```diff
@@ -0,0 +1,141 @@
+use holochain_container_api::{
+    config::{load_configuration, Configuration},
+    container::Container as RustContainer,
+};
+use holochain_core::{
+    logger::Logger,
+    signal::{signal_channel, SignalReceiver},
+};
+use holochain_core_types::{cas::content::Address, dna::capabilities::CapabilityCall};
+use neon::{context::Context, prelude::*};
+
+use crate::config::*;
+
+#[derive(Clone, Debug)]
+struct NullLogger {}
+
+impl Logger for NullLogger {
+    fn log(&mut self, _msg: String) {}
+}
+
+pub struct NodeContainer {
+    container: RustContainer,
+    _signal_rx: SignalReceiver,
+}
+
+declare_types! {
+
+    /// A Container can be initialized either by:
+    /// - an Object representation of a Configuration struct
+    /// - a string representing TOML
+    pub class JsContainer for NodeContainer {
```

Comment:
Why this duplication of node Container? js_test_container.rs still exists.

Reply:
I didn't add that, I just made things work for what was there.

Reply:
Was this file renamed from `container.rs` to `js_test_container.rs` in develop and accidentally added here after a merge, @zippy?